app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -42,6 +43,11 @@ protected void onCreate(Bundle savedInstanceState) {
             setupDrawerContent(navigationView);
         }
 
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            getWindow().setStatusBarColor(getResources().getColor(R.color.color_primary));
```

Comment:
Can be done from the server.  Set a primary color field in the event
endpoint.
Fetch it, and setColor programmatically wherever required?
On Jun 30, 2015 10:43 PM, "Mohit Kanwal" <notifications@github.com> wrote:

> In app/src/main/java/org/fossasia/openevent/activities/MainActivity.java
> <https://github.com/fossasia/open-event-android/pull/47#discussion_r33597794>
> :
>
> > @@ -42,6 +43,11 @@ protected void onCreate(Bundle savedInstanceState) {
> >              setupDrawerContent(navigationView);
> >          }
> >
> > +        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
> > +            getWindow().setStatusBarColor(getResources().getColor(R.color.color_primary));
>
> Would depend on the event I guess. Or manually customisable by changing a
> themes.xml for different events. Or we a script to write it altogether.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fossasia/open-event-android/pull/47/files#r33597794>.
>